manager/orchestrator/update/updater_test.go

```diff
@@ -592,3 +592,195 @@ func TestUpdaterOrder(t *testing.T) {
 		}
 	}
 }
+
```

Comment:
Add a high level comment on the test-case(s) covered.

Reply:
Yes, we can do that.